032_app_L238.js

```
@@ -0,0 +1,276 @@
+
+app.get("/searchinbookingpage/:searchtext?",(req,res)=>{
+
+    console.log(req.params);
+
+    if(req.params.searchtext!=undefined){
+
+        var search = new RegExp(req.params.searchtext, 'i');
+        var searchcond = {uname :search};
+    }
+    else{
+        var searchcond = null;    
```

Biome (2.1.2)

[error] Shouldn't redeclare 'searchcond'. Consider to delete it or rename it.

'searchcond' is defined here:


(lint/suspicious/noRedeclare)